pkg/util/nodelock/nodelock.go

```diff
@@ -28,11 +28,12 @@ import (
 )
 
 const (
-	NodeLockTime = "hami.io/mutex.lock"
-	MaxLockRetry = 5
+	NodeLockTime        = "hami.io/mutex.lock"
```

Comment:
I suggest reusing `NodeLockTime` so we don't have to maintain an additional field. Reusing `NodeLockTime` will also make the logic clearer.